sidebars.js

```diff
@@ -426,19 +420,16 @@ module.exports = {
             "self-managed/concepts/access-control/permissions",
             "self-managed/concepts/access-control/roles",
             "self-managed/concepts/access-control/users",
-          ]
+          ],
         },
         "self-managed/concepts/exporters",
       ],
     },
     {
-      "Zeebe": [
+      Zeebe: [
         "self-managed/zeebe-deployment/index",
         {
           Configuration: [
-            "self-managed/zeebe-deployment/configuration/configuration",
-            "self-managed/zeebe-deployment/configuration/logging",
-            "self-managed/zeebe-deployment/configuration/gateway-health-probes",
```

Comment:
Potentially blocking:

Also several removed here. Are these old, and that's why they were removed?

Reply:
Also unintentional. Good catch